docs/configuration/configuration.md

```diff
@@ -1835,3 +1835,80 @@ tls_config:
 There is a list of
 [integrations](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage)
 with this feature.
+
+# HTTPS and authentication
```

Comment:
I think having two different configuration files explained on the same page could be confusing, it may be best to separate this out to its own page that we can reuse.